src/components/augmint-ui/button/styles.js

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+
+import { Link } from "react-router-dom";
+
+import theme from "styles/theme";
+
+const BaseButton = `
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    background-color: ${theme.colors.white};
+    color: ${theme.colors.primary};
+    cursor: pointer;
+    text-transform: uppercase;
+    font-weight: 200;
+    padding: 0 14px;
+    border-radius: 4px;
+    height: 42px;
+    line-height: 42px;
+    font-size: 13px;
+    letter-spacing: 2.6px;
+
+    :hover {
```

Comment:
isn't the `&` necessary here?

Reply:
Why is it need?
It is working fine without the `&`.

Reply:
I think it's rather a weird behaviour... following sass and less nesting conventions it should be with `&`... it's either the compiler or the browser that outsmarts us here...

Reply:
ok I understand
I fix it